packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -0,0 +1,770 @@
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The identifier for the Factor this Profile belongs to. |
+| `profileId`  | String | The Factor Profile's unique identifier.         |
+
+#### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles/{profileId}/features"
+```
+
+#### Response Types
+
+HTTP 200:
+Array of [Factor Profile Feature Object](#factor-profile-feature-object)
+
+### Get a Factor Profile Feature
+
+<ApiOperation method="get" url="/api/v1/org/factors/${factorName}/profiles/${profileId}/features/${featureId}" />
+
+#### Request Parameters
+
+| Parameter    | Type   | Description                                     |
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The identifier for the Factor this Profile belongs to. |
+| `featureId`  | String | The Factor Profile feature's unique identifier. |
+| `profileId`  | String | The Factor Profile's unique identifier.         |
+
+#### Request Example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/org/factors/{factorName}/profiles/{profileId}/features/{featureId}"
+```
+
+#### Response Types
+
+HTTP 200:
+Along with one of the Factor Profile Feature Objects:
+
+todo
+
+
+
+### Update a Factor Profile Feature
+
+<ApiOperation method="put" url="/api/v1/org/factors/${factorName}/profiles/${profileId}/features/${featureId}" />
+
+#### Request Path Parameters
+
+| Parameter    | Type   | Description                                     |
+| ------------ | ------ | ----------------------------------------------- |
+| `factorName` | String | The identifier for the Factor this Profile belongs to. |
+| `featureId`  | String | The Factor Profile feature's unique identifier. |
+| `profileId`  | String | The Factor Profile's unique identifier.         |
+
+#### Request Body (Adoption)
+
+| Property                              | Type   | Description                                                                                                 |
+| ------------------------------------- | ------ | ----------------------------------------------------------------------------------------------------------- |
+| `cardinality.max`                     | Number | The maximum number of factor instances user is allowed to enroll in.                                        |
+| `cardinality.min`                     | Number | The minimum number of factor instances user must enroll in.                                                 |
+| `selfService.eligibility`             | String | Indicates if factor may be adopted. Possible values: `ALLOWED`, `NOT_ALLOWED`                               |
```

Comment:
what does factor may be `adopted` mean?
you mean if factor can be `enrolled`

Reply:
OK, so first of all, MFA-Platform added this feature, so I'd expect _you_ to tell _me_.   If you want my opinion we should use "enrollment" everywhere instead of "adoption".  Which kind of makes the feature name seem silly, but too late now.